tools/Assembly/EggNOG/eggNOG/eggnog.cwl

```diff
@@ -13,10 +13,10 @@ hints:
 
 requirements:
   ResourceRequirement:
-    ramMin: 10000
+    ramMin: 1000
     coresMin: 32
```

Comment:
do we need 32 cores?

Reply:
We are only using 16.. so no. Good catch!